compiler/codegen/symbol.cpp

```diff
@@ -56,6 +54,10 @@
 #include <inttypes.h>
 #include <stdint.h>
 
+#ifdef HAVE_LLVM
+#include "llvm/IR/Attributes.h"
+#endif
```

Comment:
This is OK. I'd probably put the LLVM include before the first system include (`<algorithm>`). compiler/include/llvmUtil.h might work too but it's probably smarter to just put the include you need for this file here. Speaking of that, I'm surprised that the other llvm:: stuff in this file worked addFnAttr didn't. Are you sure you can't just remove the new include entirely?